src/index.ts

```diff
@@ -347,6 +353,53 @@ class Spanner extends GrpcService {
     this.directedReadOptions = directedReadOptions;
   }
 
+  /**
+   * Gets the InstanceAdminClient object.
+   * The InstanceAdminClient object is also shared for handwritten client and should not be manually closed.
+   * @returns {v1.InstanceAdminClient} The InstanceAdminClient object
+   * @example
+   *  ```
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner({
+   *    projectId: projectId,
+   *  });
+   * const instanceAdminClient = spanner.get_instance_admin_client();
+   * ```
+   */
+  get_instance_admin_client(): v1.InstanceAdminClient {
+    const clientName = 'InstanceAdminClient';
+    if (!this.clients_.has(clientName)) {
+      this.clients_.set(
+        clientName,
+        new v1[clientName](this.options as ClientOptions)
+      );
+    }
+    return this.clients_.get(clientName)! as v1.InstanceAdminClient;
+  }
+  /**
```

Comment:
nit: add an empty line between the end of the last method and the start of this method